packages/blockchain/src/dbManager.ts

```diff
@@ -175,7 +202,7 @@ export default class DBManager {
   /**
    * Performs a batch operation on db.
    */
-  batch(ops: Array<any>): Promise<any> {
-    return this._db.batch(ops)
+  batch(ops: DBOp[]) {
```

Comment:
oh it is async, i forgot to add the keyword, i just removed the return type because it can automatically interpret it:

<img width="736" alt="Screen Shot 2020-08-17 at 11 39 05 AM" src="https://user-images.githubusercontent.com/22116/90431771-65676f00-e07e-11ea-81f8-41f98350fa7d.png">

Reply:
added in https://github.com/ethereumjs/ethereumjs-vm/pull/833/commits/e6e6487c11af6189741b388ea8968af1c1e0016d